packages/client/src/__tests__/integration/errors/incorrect-column-type/test.ts

```diff
@@ -20,8 +20,9 @@ test('incorrect-column-type', async () => {
             12   ],
             13 })
             14 
-          → 15 await expect(prisma.user.findMany()).rejects.toThrowErrorMatchingInlineSnapshot(
-            Error converting field "name" of expected non-nullable type "String", found incompatible value of "123".
+          → 15 await expect(prisma.user.findMany()).rejects.toThrowErrorMatchingInlineSnapshot(\`
```

Comment:
That additional backtick is intended here?

Reply:
I think there was another off by 1 error in original code. If there is no `{` in the line, it cut of last character, where the intent was, I think, was to keep the entire line unchanged.
But on the other hand, this snippet is wrong in both "before" and "after" snapshots. It should've ended at `await expect(prisma.user.findMany(`, I'll see if I can quickly fix it.

Reply:
Update: "quickly = 1.5 month" 🤣

Reply:
Don't want to know what "soon" would mean then :D